src/crl.c

```diff
@@ -1022,8 +1022,8 @@ int LoadCRL(WOLFSSL_CRL* crl, const char* path, int type, int monitor)
             }
         }
 
-        if (!skip && ProcessFile(NULL, name, type, CRL_TYPE, NULL, 0, crl)
-                                                           != WOLFSSL_SUCCESS) {
+        if (!skip && ProcessFile(NULL, name, type, CRL_TYPE, NULL, 0, crl,
+                                 NO_VERIFY) != WOLFSSL_SUCCESS) {
```

Comment:
Always NO_VERIFY, why?

Reply:
Its misleading because the `CRL_TYPE` flow gets loaded using `BufferLoadCRL`, which has a zero (0) indicate NO_VERIFY. The LoadCRL is loading CRL's to trust, so no "verify" is required. For CRL that means allowing it to bypass the `ASN_CRL_NO_SIGNER_E` error.  I pushed some cleanups around the CRL code to help.

Reply:
I don't think that's right.  The last parameter to BufferLoadCRL is noVerify.  A zero(0) to noVerify means verify doesn't it?  Because a 1 to noVerify would mean don't Verify I would think.

Reply:
You are so right! I had it backwards and Jenkins caught it too. Good thing we are cleaning this up.

Reply:
Fix force pushed.